<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuDialog</class>
 <widget class="QDialog" name="MenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1762</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>40</number>
     </property>
     <item row="1" column="2">
      <widget class="QPushButton" name="englishButton">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">English</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Language</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="frenchButton">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Fran√ßais</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="EGlineColorButton">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Change lines' color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="EGresetButton">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reset Levels</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="quitButton">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Eulerian Graphs</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="aboutButton">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QPushButton" name="sourcesButton">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sources page on github.com...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
